### ISSUE TITLE
Add missing `types` export for esm and re-export type `SvgLoaderOptions`

### ISSUE THREAD

**Issue by dsvgl** [cpsoinos/nuxt-svgo#267]
I wanted to use exported `SvgLoaderOptions` type in my custom module like this.

```js
import type { SvgLoaderOptions } from 'nuxt-svgo';
```

But TS complains:

`
There are types at /Users/xxx/code/my-repo/packages/nuxt-modules/node_modules/nuxt-svgo/dist/module.d.ts', but this result could not be resolved when respecting package.json "exports". The nuxt-svgo' library may need to update its package.json or typings.
`

There are types defined in nuxt-svgo package.json for cjs with this line: `"types": "./dist/module.d.ts"`.
The `types` property under the `exports` key is missing:

```json
"exports": {
  ".": {
    "import": "./dist/module.mjs",
    "require": "./dist/module.cjs",
    "types": "./dist/module.d.ts"  // <------- add this
  }
},
```

I tested that this solves it by modifying package.json inside node_modules.

With this, I can import the type `ModuleOptions` from nuxt-svgo. But not type `SvgLoaderOptions`.
Could you also re-export `SvgLoaderOptions` within your module.ts like so (not totally sure if this would work)?

```ts
import { SvgLoaderOptions, svgLoader } from './loaders/vite'
export { SvgLoaderOptions } from './loaders/vite' // <------- add this
```

But I can also work with type `ModuleOptions` by index-accessing it with `autoImportPath`.

```ts
let nuxtSvgoModuleAutoImportPath: ModuleOptions['autoImportPath'] = false;
```

**Comment by jd1378** [comment]
Hi
sure, will do in a moment

**Comment by jd1378** [comment]
released in [v4.0.1](https://github.com/cpsoinos/nuxt-svgo/releases/tag/v4.0.1)

**Comment by dsvgl** [comment]
Thank you! 💖
